_inc/client/writing/widgets.jsx

```diff
@@ -24,42 +31,46 @@ class Widgets extends Component {
 				module="widgets"
 				hideButton
 			>
-				<SettingsGroup
-					module={ { module: 'widgets' } }
-					support={ {
-						text: this.props.widgetsModule.description,
-						link: 'https://jetpack.com/support/extra-sidebar-widgets/',
-					} }
-				>
-					<ModuleToggle
-						slug="widgets"
-						disabled={ ! isLinked }
-						activated={ this.props.widgetsActive }
-						toggling={ this.props.isSavingAnyOption( 'widgets' ) }
-						toggleModule={ this.props.toggleModuleNow }
+				{ foundWidgets && (
+					<SettingsGroup
+						module={ { module: 'widgets' } }
+						support={ {
+							text: this.props.widgetsModule.description,
+							link: 'https://jetpack.com/support/extra-sidebar-widgets/',
+						} }
 					>
-						{ __(
-							'Make extra widgets available for use on your site including subscription forms and Twitter streams'
-						) }
-					</ModuleToggle>
-				</SettingsGroup>
-				<SettingsGroup
-					module={ { module: 'widget-visibility' } }
-					support={ {
-						text: __( 'Configure widgets to appear only on certain posts or pages.' ),
-						link: 'https://jetpack.com/support/widget-visibility/',
-					} }
-				>
-					<ModuleToggle
-						slug="widget-visibility"
-						disabled={ ! isLinked }
-						activated={ this.props.widgetVisibilityActive }
-						toggling={ this.props.isSavingAnyOption( 'widget-visibility' ) }
-						toggleModule={ this.props.toggleModuleNow }
+						<ModuleToggle
+							slug="widgets"
+							disabled={ ! isLinked }
+							activated={ this.props.widgetsActive }
+							toggling={ this.props.isSavingAnyOption( 'widgets' ) }
+							toggleModule={ this.props.toggleModuleNow }
+						>
+							{ __(
+								'Make extra widgets available for use on your site including subscription forms and Twitter streams'
```

Comment:
Can you change this to:
> Enable widget visibility controls to display widgets only on particular posts or pages. (Toggle)

Reply:
I believe you meant the toggle for Widget Visibility, I've updated that label.